metagraph/src/graph/alignment/alignment.cpp

```diff
@@ -489,11 +871,10 @@ Json::Value Alignment::path_json(size_t node_size, std::string_view label) const
 }
 
 Json::Value Alignment::to_json(std::string_view full_query,
-                               const DeBruijnGraph &graph,
+                               size_t node_size,
                                bool is_secondary,
                                std::string_view read_name,
                                std::string_view label) const {
```

Comment:
Not clear what is `full_query` and how it relates to `query_`.

Reply:
`full_query` can actually be reconstructed from the alignment, so I changed this function